drivers/volume/portworx/portworx.go

```diff
@@ -659,26 +659,28 @@ func (p *portworx) GetClusterID() (string, error) {
 }
 
 func (p *portworx) OwnsPVCForBackup(coreOps core.Ops, pvc *v1.PersistentVolumeClaim, cmBackupType string, crBackupType string) bool {
-	return p.OwnsPVC(coreOps, pvc)
+	return p.IsSupportedPVC(coreOps, pvc, true)
 }
 
 func (p *portworx) OwnsPVC(coreOps core.Ops, pvc *v1.PersistentVolumeClaim) bool {
-
+	return p.IsSupportedPVC(coreOps, pvc, false)
+}
+func (p *portworx) IsSupportedPVC(coreOps core.Ops, pvc *v1.PersistentVolumeClaim, skipFBVols bool) bool {
```

Comment:
skibFBVols could use an explanatory comment on _why_ we'd want to skip them
nit: add space between functions

Reply:
Lets rename the variable to `skipDirectAccessVolumes`.  This boolean would be used for FA, FB and proxy volumes

Reply:
sounds good